tests/test_grains.py

```diff
@@ -480,7 +555,7 @@ def test_find_grains(
                         [
                             [1, 1, 1, 1, 1, 1, 1, 1, 1],
                             [1, 0, 0, 0, 1, 0, 1, 0, 1],
-                            [1, 0, 0, 0, 1, 0, 1, 1, 1],
+                            [1, 0, 1, 0, 1, 0, 1, 1, 1],
```

Comment:
Any idea why there is a this single pixel change?

Reply:
Yeah in my new code, I add an `update_background_class` call at the end of `find_grains` which ensures that the background mask is correct. Previously I did not do this, and the background mask erroneously did not add that background pixel back in. This change in the test reflects the updated behavior.

Reply:
Ok cool, always good to improve things :+1: